src/test/scala/io/viash/platforms/nextflow/NextflowScriptTest.scala

```diff
@@ -135,6 +135,18 @@ class NextflowScriptTest extends AnyFunSuite with BeforeAndAfterAll {
     assert(exitCode == 0, s"\nexit code was $exitCode\nStd output:\n$stdOut\nStd error:\n$stdErr")
   }
 
+  test("Test whether aliasing works", DockerTest, NextflowTest) {
+    val (exitCode, stdOut, stdErr) = NextflowTestHelper.run(
+      mainScript = "target/nextflow/test_wfs/alias/main.nf",
+      args = List(
+        "--publish_dir", "output"
+      ),
+      cwd = tempFolFile
+    )
+
+    assert(exitCode == 0, s"\nexit code was $exitCode\nStd output:\n$stdOut\nStd error:\n$stdErr")
```

Comment:
Suggestion: check the process name in stdout?

Reply:
This is already covered by the assertions in the main.nf:

```groovy
  assert step1.getName() == "step1"
  assert step1_alias.getName() == "step1_alias"
```

(checks whther the workflow was named `step1` and `step1_alias`)

And

```groovy
      assert data.output1.getFileName().toString() == "one.step1.output.txt"
      assert data.output2.getFileName().toString() == "one.step1_alias.output.txt"
```

(checks whether the `key` in `.run()` was set properly)

However, I guess it couldn't hurt to check the stdout as well.